src/components/Todo/index.jsx

```diff
@@ -9,9 +9,16 @@ import styles from './styles.css';
 const Todo = ({ onClick, completed, target, callToAction }) => (
   <li onClick={onClick}
       styleName={completed ? 'todo-completed' : 'todo'}>
-      Call <PhoneNumber value={target} />:
+    <PhoneNumber value={target} actionId={callToAction.uid} />
+    <div styleName={completed ? 'todo-completed-summary' : 'todo-summary'}>
       {callToAction.data['call-to-action.title'].value[0].text}
-      <CallToAction {...callToAction.data} uid={callToAction.uid} key={callToAction.uid} />
+    </div>
+    {!completed &&
+      <CallToAction {...callToAction.data}
+                    uid={callToAction.uid}
+                    key={callToAction.uid}
+                    inTodo={true} />
```

Comment:
I *really* don't like inventing this `inTodo` parameter, but I can't come up with a better way to make the CallToAction and its descendent PhoneNumbers behave differently when it's inside a Todo.  (Specifically I want to hide the CTA's heading, time, and phone-numbers.  I tried doing that with CSS Modules but that doesn't seem to be possible.)

Would I be better off not trying to reuse the CallToAction component here, and just repeating some of the JSX and CSS code around script & notes?